Modules/pyexpat.c

```diff
@@ -1921,40 +1930,42 @@ pyexpat_exec(PyObject *mod)
     MYCONST(XML_PARAM_ENTITY_PARSING_ALWAYS);
 #undef MYCONST
 
-    static struct PyExpat_CAPI capi;
+    struct PyExpat_CAPI *capi = PyMem_Calloc(1, sizeof(struct PyExpat_CAPI));
```

Comment:
You need to check for memory allocation failure (check for NULL and call PyErr_NoMemory).

Reply:
Thanks, done.